anki/web/editor.js

```diff
@@ -119,8 +120,11 @@ var EditorCloset = {
     setupOcclusionEditor: async (closet, maxOcclusions) => {
         const fieldElements = [];
 
-        for (const field of instances[0].fields) {
-            fieldElements.push(await get(field.editingArea.editingInputs).find((input) => input.name === "rich-text").element);
+        for (const field of EditorField.instances) {
+            const richTextInputAPI = get(field.editingArea.editingInputs)
+                .find((input) => input.name === "rich-text");
+            subscriptionCallbacks.push(richTextInputAPI.preventResubscription());
```

Comment:
This line was meant to be included in https://github.com/hgiesel/closet/pull/95/commits/a0f52a95e7099407c0d4c5299cff8fd14789c7df.